ext/ldap/ldap.stub.php

```diff
@@ -4,177 +4,177 @@
 
 #ifdef HAVE_ORALDAP
 /** @return resource|false */
-function ldap_connect(string $hostname = UNKNOWN, int $port = 389, string $wallet = UNKNOWN, string $wallet_passwd = UNKNOWN, int $authmode = GSLC_SSL_NO_AUTH) {}
+function ldap_connect(string $uri = null, int $port = 389, string $wallet = UNKNOWN, string $wallet_passwd = UNKNOWN, int $authmode = GSLC_SSL_NO_AUTH) {}
```

Comment:
```suggestion
function ldap_connect(?string $uri = null, int $port = 389, string $wallet = UNKNOWN, string $wallet_passwd = UNKNOWN, int $authmode = GSLC_SSL_NO_AUTH) {}
```
and elsewhere. These need to be explicitly marked nullable. I thought the script would throw an error for that :/

Reply:
I think it do throws (at least I got such an exception a few weeks ago)

Reply:
Oh, ok.

Should I use ?string or string|null ? I think I’ve seen commits changing one for the other which is why I ask.

Reply:
Now, we exclusively use `?` for nullable types where it is possible, so please use this format.